pom.xml

```diff
@@ -23,7 +23,7 @@
   </modules>
 
   <properties>
-    <rhba.images.version.prefix>77</rhba.images.version.prefix>
+    <rhba.images.version.prefix>78</rhba.images.version.prefix>
```

Comment:
This should be ideally placed as a separate PR, as it is not directly related to the original purpose of PR.
But in this case you can keep it here.

Reply:
I know. I'm running the jobs in templates and operator to ensure this is working fine before merging. (I updated the BXMS repo as well). I will revert it afterwards.

Reply:
ok

Reply:
reverted